tests/test1119.pl

```diff
@@ -182,13 +182,13 @@ sub scanman_md_dir {
     }
     if($doc{$e}) {
         if($verbose) {
-            print $e."\n";
```

Comment:
historical note: perltidy complained about "operator in possible indirect object location." which is related to usage of a **period** ... this is fixed by replacing with **comma**.